package.json

```
@@ -10,6 +10,9 @@
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "dependencies": {
+    "@apollo/client": "^3.6.9",
+    "@react-native-async-storage/async-storage": "^1.17.10",
+    "graphql": "^16.6.0",
```

Nós costumamos fixar a versão das libs instaladas para não ter chance de rolar problema de compatibilidade. 
```suggestion
    "graphql": "16.6.0",
```